migration/run.js

```diff
@@ -1,29 +1,64 @@
 const hasUntilDestroy = /import\s*{\s*[^}]*untilDestroyed[^}]*}\s*from\s*("|')ngx-take-until-destroy\1(?=[^]*untilDestroyed\(\w*\)[^]*)/;
 const catchImport = /import\s*{\s*[^}]*untilDestroyed[^}]*}\s*from\s*("|')ngx-take-until-destroy['|"];/;
+const catchOnDestroy = /\s*(public\s+)?ngOnDestroy\s*[(]\s*[)](\s*:\s*void)?\s*[{]\s*[}]/;
 
 const glob = require('glob');
 const fs = require('fs');
 
-const base = `src/app`;
+const base = `app`;
+
+let removeOnDestroy = false;
+
+glob(`${base}/**/*.ts`, {}, function(_, files) {
+  if (process.argv.includes('--removeOnDestroy')) removeOnDestroy = true;
 
-glob(`${base}/**/*.ts`, {}, function(er, files) {
   files.forEach(path => {
-    fs.readFile(path, 'utf8', function(err, text) {
+    fs.readFile(path, 'utf8', function(_, text) {
       if (hasUntilDestroy.test(text)) {
         console.log(`Replaced ${path}`);
-        const result = text
+        let result = text
           .replace(
             /((?:@\w*\([^]*\)[\n\s\r\t]*)(?=([\n\s\r\t]*export[\s\r\t]*class))\2)/,
             '@UntilDestroy()\n$1'
           )
           .replace(
             catchImport,
-            `import { untilDestroyed, UntilDestroy } from '@ngneat/until-destroy';`
+            `import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';`
           );
+
+        if (removeOnDestroy) result = cutOnDestroyHook(result);
         fs.writeFile(path, result, 'utf8', function(err) {
           if (err) return console.log(err);
         });
       }
     });
   });
 });
+
+function cutOnDestroyHook(text) {
```

Comment:
Can you please add some tests?

Reply:
Ok. In `migration` folder?

Reply:
Yes, thanks!

Reply:
@NetanelBasal How can I test this script? It hasn't `module.exports`. It's possible testing like this

Reply:
@NetanelBasal How can I test this script? It hasn't module.exports. Possible solution:
```js
it(example, (done) => {
  require("./run.js");
  setTimeout(done, 5000)
}
```
But it's rude test and only single-file script - i can't split it for better testability. What do you think about it?

Reply:
Yes, we should refactor the code to make it testable.